### PR TITLE
first widget test for `graphql_flutter` package

### PR DESCRIPTION
**Pull request by truongsinh**
very first widget test for `graphql_flutter` package. When doing so, I happen to discover that we are not null-safe in some places

### CODE REVIEW

File: packages/graphql/pubspec.yaml [hunk 1]
```diff
@@ -1,7 +1,7 @@
 name: graphql
 description: A stand-alone GraphQL client for Dart, bringing all the features from
   a modern GraphQL client to one easy to use package.
-version: 1.0.2-beta.2
+version: 1.1.0
```
Comment (HofmannZ) [inline packages/graphql/pubspec.yaml]: Can you please change this back?

Comment (truongsinh) [reply packages/graphql/pubspec.yaml]: ok, sorry i did't pay attention to this.

Comment (truongsinh) [reply packages/graphql/pubspec.yaml]: ah no, I meant to merge this to `master`, not `beta`. It seems that I can't change the destination, so I will close this and make another PR.

**Comment by mainawycliffe** [comment]
We should be making this changes against beta, not master. But looks good. https://github.com/zino-app/graphql-flutter/blob/master/CONTRIBUTING.md

**Comment by truongsinh** [comment]
> We should be making this changes against beta, not master. But looks good. https://github.com/zino-app/graphql-flutter/blob/master/CONTRIBUTING.md

Sorry I took a long gap 😅. Anyway, I don't understand why we should target `beta` branch (I'm a fan of https://trunkbaseddevelopment.com/), but let's take that discussion in another thread. I will make neccessary change.

**Comment by mainawycliffe** [comment]
LGTM

**Pull request merged**